.pre-commit-config.yaml

```diff
@@ -49,3 +49,12 @@ repos:
     entry: ./bqetl
     args: [format, --check]
     types: [sql]
+- repo: local
+  hooks:
+  - id: bqetl_backfill_validate
+    name: bqetl_backfill_validate
+    language: script
+    entry: ./bqetl
+    args: [backfill, validate]
```

Comment:
I think this needs `--ignore-missing-metadata` otherwise it will fail if you don't have all the generated sql

Reply:
ah, that's great. We should add that then. To be honest, I missed the fact this option exists.